packages/fxa-customs-server/test/remote/too_many_bad_logins.js

```diff
@@ -108,48 +108,48 @@ test('too many failed logins from the same email', async (t) => {
   const IP3 = '192.0.2.13';
 
   // eslint-disable-next-line no-unused-vars
```

Comment:
Q: Does this still need the `// eslint-disable-next-line no-unused-vars` now that we have a sparse array?